api/spec/controllers/spree/api/checkouts_controller_spec.rb

```diff
@@ -308,7 +308,9 @@ module Spree
       context "#{action}" do
         context "with order in confirm state" do
           subject do
-            api_put action, params
+            ActiveSupport::Deprecation.silence do
```

Comment:
Ok, reading the comment above, we only want to silence for the `:next` case, not the `:complete` case, right? Is that too much of a pain to accomplish?